test/web-platform-tests/to-upstream/html/semantics/forms/the-input-element/default-type-dont-upstream.html

```diff
@@ -1,33 +1,87 @@
 <!DOCTYPE html>
-<meta charset="utf-8" />
+<meta charset=utf-8>
 <!-- This test should not be upstreamed, as it tests the same thing as html/dom/reflection-forms.html. -->
```

Comment:
I'm not sure this is true anymore as I don't think the reflection helper tests Ks and Ss.

This also doesn't seem to test Ss, or test all the possible cases, just one with week, hmm.

Reply:
It looks like `"ſ".toLowerCase()` is still `"ſ"`, but I added U+0130 LATIN CAPITAL LETTER I WITH DOT ABOVE (İ).

Reply:
Also tested all possible cases.